internal/rbd/nodeserver.go

```diff
@@ -413,7 +413,23 @@ func (ns *NodeServer) stageTransaction(
 	}
 
 	if volOptions.isEncrypted() {
+		if volOptions.Mounter == rbdNbdMounter {
+			stdout, stderr, saveErr := util.ExecCommand(ctx, "ps", "aux | grep rbd-nbd")
```

Comment:
I do not think you can use `| grep` in `ExecCommand()`, use something like this instead?

```go
stdout, stderr, saveErr := util.ExecCommand(ctx, "sh", "-c", "ps aux | grep rbd-nbd")
```

Reply:
Got it, Thanks @nixpanic